src/renderer-process/js/video-layer-view.js

```diff
@@ -71,9 +79,24 @@ fluid.defaults("bubbles.videoLayerView", {
             args: "{that}.video.element"
         },
 
+        "onCreate.renderModulationMatrixContainer": {
+            funcName: "bubbles.videoLayerView.renderModulationMatrixContainer",
+            args: "{that}"
+        },
+
         "onVideoAdded.updateVideoURL": {
             changePath: "video.url",
             value: "{arguments}.0.0"
         }
+    },
+
+    markup: {
+        modulationMatrixContainer: "<div class='bubbles-modulation-matrix'></div>"
     }
 });
+
+bubbles.videoLayerView.renderModulationMatrixContainer = function (that) {
+    var modulationMatrixContainer = $(that.options.markup.modulationMatrixContainer);
```

Comment:
See above for suggestions on "containerRenderingView" which will eliminate some complexity and an extra event